samples/deviceCodeSample/src/main/java/deviceCodeFlowMain.java

```diff
@@ -0,0 +1,51 @@
+import com.azure.identity.DeviceCodeCredential;
+import com.azure.identity.DeviceCodeCredentialBuilder;
+import com.microsoft.graph.authentication.TokenCredentialAuthProvider;
+import com.microsoft.graph.exceptions.AuthenticationException;
+import com.microsoft.graph.httpcore.HttpClients;
+import okhttp3.*;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+
+public class deviceCodeFlowMain {
+
+    //Replace CLIENT_ID with your own client id from an adequately configured app
+    //for requirements visit:
+    //https://github.com/Azure/azure-sdk-for-java/wiki/Set-up-Your-Environment-for-Authentication#enable-applications-for-device-code-flow
+    private final static String CLIENT_ID = "cfb5d84c-c88f-466c-81fb-0fe9fa8da052";
+
+    //Set the scopes for your ms-graph request
+    private final static List<String> SCOPES = Arrays.asList("user.ReadBasic.All", "User.Read");
+
+    public static void main(String[] args) throws AuthenticationException {
+        deviceCodeFlow();
```

Comment:
opinion: seems like a redundant function call. Class and file names are already self-documenting, I think having a shorter sample code is better.

Reply:
fixed